test/controllers/property.controller.test.js

```diff
@@ -1452,7 +1492,7 @@ describe('Property Controller', () => {
         await addProperty(vendorProperty);
       });
 
-      context('when unknown filter is used', () => {
+      context('Unknown Filters', () => {
```

Comment:
This should be a `describe` block with the updates.

```suggestion
	      describe('Unknown Filters', () => {
```